src/Microsoft.AspNetCore.Routing/CompositeEndpointDataSource.cs

```diff
@@ -56,7 +56,7 @@ private void EnsureInitialized()
             }
         }
 
-        // Note: we can't use DataSourceDependantCache here because we also need to handle a list of change
+        // Note: we can't use DataSourceDependentCache here because we also need to handle a list of change
```

Comment:
This was using an older and more britisher spelling. We use American English because @Eilon says. I think I was the one who messed it up anyway.

Reply:
Well if you **want** your English to be wrong...

![image](https://user-images.githubusercontent.com/303201/42859066-0634be32-8aa6-11e8-8c13-282f6cd865c1.png)